core/nginx/letsencrypt.py

```diff
@@ -4,10 +4,16 @@
 import time
 import subprocess
 
+hostnames = list(set(os.environ['HOSTNAMES'].split(',')))
+for hostname in hostnames:
+    if not hostname.startswith('autoconfig.'):
+        hostnames.append(f'autoconfig.{hostname}')
+hostnames = ','.join(set(hostnames))
```

Comment:
Not sure if this will work... what we really want is a list of domains, as configured in the DB

Reply:
As discussed, this isn't required